docs/src/overview/structure.md

```diff
@@ -1,77 +1,88 @@
 ---
-title: "Structure"
+title: Structure
 weight: 1
+description: Learn about how your Platform.sh environments are structured and which files control that structure.
 ---
 
-Every application you deploy on Platform.sh is built as a **virtual cluster**,
-containing a set of containers.
-The default branch of your Git repository is always deployed as the production cluster.
-Any other branch can be deployed as a development cluster.
+{{< note >}}
 
-By default, you can have up to three live development clusters at once,
-but you can buy more on a per-project basis.
+This page describes how things work on Grid projects.
+[Dedicated Gen 3](../dedicated-gen-3/overview.md) projects are similar,
+but they run on dedicated virtual machines and each container is replicated three times.
 
-There are three types of containers within your cluster:
+For Dedicated projects, read about how [Dedicated projects are structured](../dedicated/overview/_index.md)
 
-* one *router*
-* one or more *application* containers
-* zero or more *service* containers
+{{< /note >}}
 
-All of those containers are managed by three special files in your Git repository:
+Each environment you deploy on Platform.sh is built as a set of containers.
+Each container is an isolated instance with specific resources.
 
-* `.platform/routes.yaml`
-* `.platform/services.yaml`
-* `.platform.app.yaml`
+Each environment has 2 to 4 types of containers:
 
-In most cases, that means your repository looks like this:
+* One [*router*](#router) (configured in a `.platform/routes.yaml` file)
+* One or more [*app* containers](#apps) (configured in `.platform.app.yaml` files)
+* Zero or more [*service* containers](#services) (configured in a `.platform/services.yaml` file)
+* Zero or more [*worker* containers](#workers) (configured in the files for apps)
+
+If you have two app containers, two services (a database and a search engine), and a worker,
+requests to your environment might look something like this:
+
+![A user request goes to the router, which sends it to either a Node.js app or a Python app. Each app communicates separately with the database and search services and sends responses to the user. The Node.js app triggers actions in a worker, which communicates separately with the database.](/images/config-diagrams/structure-diagram.png)
```

Comment:
:bulb:
For future updates to the diagram and to keep the look and feel, it might be nice to add a comment to know how the image was generated and how to update it.
Or maybe a more generic info on the [markup-format](https://github.com/platformsh/platformsh-docs/blob/main/contributing/markup-format.md#images).

Reply:
The diagram is done using internal processes. See the Context for the link where it can be found. I tried Mermaid, but it was harder to get it forced into a straight line.